packages/contracts/CHANGELOG.md

```diff
@@ -26,21 +26,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed unused `ERC1271Mock` contract.
 - Removed the `setSignatureValidator` function and `signatureValidator` variable in `DAO`. In places, where the function must remain to not alter the `IDAO` interface ID, it will revert and explanatory notes are put in place..
 
-## v1.3.1-rc0
-
-### Added
+## v1.3.1
```

Comment:
We should release this as `aragon/osx@1.3.1` NPM package after we release `aragon/osx@1.3.0`.